Bot.js

```diff
@@ -4,59 +4,73 @@ const client = new Discord.Client();
 
 //Constant for commands file
 const {commands} = require('./src/');
-
-var commandChannels = [];
+const {prefix, token} = require('./config/Bot.json')
+const blacklist = [];
```

Comment:
This is a blacklist of channel names not to use commands in right?

Reply:
Correct, instead of names it uses channel IDs